enumerables.rb

```diff
@@ -1,59 +1,60 @@
+# frozen_string_literal: true
+
+# enumerable methods
 module Enumerable
   def my_each
     arr = self
-      for i in arr
-        yield(i)
-      end
+    arr.each do |i|
+      yield(i)
+    end
   end
 
   def my_each_with_index
     arr = self
-      for i in arr
-        v = arr.find_index(i)
-        yield(i, v)
-      end
+    arr.each do |i|
+      v = arr.find_index(i)
+      yield(i, v)
+    end
   end
 
   def my_select
     arr = self
     result = []
-    for i in arr
-      if  yield i
-        result.push(i)
-        end
+    arr.each do |i|
+      result.push(i) if yield i
     end
     result
   end
 
-  def my_all(argument = nil)
+  def my_all(_argument = nil)
     arr = self
     if block_given?
       arr.my_each do |i|
-      return false unless yield(i)
+        return false unless yield(i)
       end
     else
       my_each { |i| return false if i.nil? || i == false }
     end
     true
   end
 
-  def my_all(argument = nil)
+  def my_any(_argument = nil)
```

Comment:
- [ ] `any?` method can take a class name as argument to check if all belong to that class. For example:
```ruby
[nil, 3, 5].any? Integer
# should return: true
# your implementation returns false.
```
Let's fix it.